src/sentry/utils/snuba_rpc.py

```diff
@@ -143,7 +151,7 @@ def create_subscription(req: CreateSubscriptionRequest) -> CreateSubscriptionRes
     cls = req.__class__
     endpoint_name = cls.__name__
     class_version = cls.__module__.split(".", 3)[2]
-    http_resp = _make_rpc_request(endpoint_name, class_version, req)
+    http_resp = _make_rpc_request(endpoint_name, class_version, None, req)
```

Comment:
This will strip the referrer from the autocompletion RPCs in the mean time. Is there a reason why it has to be passed here instead of inferred inside `_make_rpc_request`?

Reply:
Wait it will? isn't autocompletion calling `rpc`? I prefer this since its clearer that not passing a referrer is expected behaviour for `create_subscription`

Reply:
Oh my bad 🤦 I was looking at the wrong function. Ignore